src/tiles/grid/styles/tile-grid-shared-styles.ts

```diff
@@ -0,0 +1,98 @@
+import { css } from 'lit';
+
+/**
+ * Base tile styles
+ */
+
+const tileCornerRadius = css`var(--collectionTileCornerRadius, 4px)`;
+const tileBackgroundColor = css`var(--TileBackgroundColor, #ffffff)`;
```

Comment:
Can we make this `--tileBackgroundColor` (lowercase t) for consistency?

Reply:
+1